lambda/storeHistoricalDataInDB.ts

```diff
@@ -0,0 +1,44 @@
+import {
+	type TimestreamWriteClient,
+	type _Record,
+	WriteRecordsCommand,
+} from '@aws-sdk/client-timestream-write'
+import { chunkArray } from './chunkArray.js'
+
+export const storeHistoricalDataInDB =
+	({
+		tsw,
+		dbName,
+		tableName,
+	}: {
+		tsw: TimestreamWriteClient
+		dbName: string
+		tableName: string
+	}) =>
+	async (
+		records: _Record[],
+	): Promise<{ success: boolean } | { error: Error }> => {
+		const recordsToTimestream = chunkArray({
+			array: records,
+			chunkSize: 100,
+		}) as _Record[][]
+		for (const rec of recordsToTimestream) {
+			if (rec.length == 0) {
+				console.log('Empty record!')
```

Comment:
**problem**: no use of console in library functions. Use an optional logger as an argument if you need this information.